cathpy.core.error
=================

.. code:: python

    from cathpy.core import error as err

    raise err.OutOfBoundsError('error message')

.. automodule:: cathpy.core.error
    :members:

